src/pivot-table/hooks/use-measure-text.ts

```diff
@@ -13,7 +13,7 @@ export interface MeasureTextHook {
   measureText: (text: string) => number;
 }
 
-const MAGIC_DEFAULT_CHAR = "M";
+const MAGIC_DEFAULT_CHAR = "N";
```

Comment:
This will change with https://github.com/qlik-oss/sn-pivot-table/pull/388

Reply:
Ah, think I changed it to test something, will revert